.github/workflows/labeler.yml

```diff
@@ -0,0 +1,16 @@
+# Workflow to associate labels automatically
+name: labeler
+# Trigger the workflow on pull request events
+on: [pull_request]
+jobs:
+  label:
+    runs-on: ubuntu-18.04
```

Comment:
why not using latest Ubuntu? `ubuntu-20.04`

Reply:
Because those who made the script use that one, dno

Reply:
shall we try latest?

Reply:
why not ^^